draft-ietf-add-ddr.md

```diff
@@ -342,6 +345,21 @@ Resolver owners that support authenticated discovery will need to list valid
 referring IP addresses in their TLS certificates. This may pose challenges for
 resolvers with a large number of referring IP addresses.
 
+## Server Name Handling
+
+Clients MUST NOT use "resolver.arpa" as the server name either in the TLS
+Server Name Indication in TLS ({{?RFC8446}}) for DoT or DoH connections,
+or in the URI host for DoH requests.
+
+Designated DoH resolvers that support authenticated discovery for clients
+that only know an IP address MUST accept both of the following as the
+URI host in requests:
```

Comment:
I'm still skeptical about this bit.

Reply:
Yes, I understand. We'll cover this at the meeting next week.

The main reason I care strongly about this is that, because the intent of checking the address in the certificate is to allow the use of other addresses, when I discover https://dns.google/dns-query from 8.8.8.8, I want to let clients access the resolver over IPv6 using the normal connection establishment logic for a DNS name dns.google; and it is actually quite awkward for many URL APIs to have the URL of the request be https://8.8.8.8/dns-query but have the connection to the name dns.google with happy eyeballs landing me on an IPv6 address. As long as the address 8.8.8.8 is in the certificate, the validation check is satisfied. I don't see merit in forcing other values, like the URI host, to be locked to that.